docs/reference/requirement-sets/excel-api-1-11-requirement-set.md

```diff
@@ -64,4 +64,4 @@ The following table lists the APIs in Excel JavaScript API requirement set 1.11.
 ## See also
 
 - [Excel JavaScript API Reference Documentation](/javascript/api/excel?view=excel-js-1.11)
-- [Excel JavaScript API requirement sets](./excel-api-requirement-sets.md)
\ No newline at end of file
+- [Excel JavaScript API requirement sets](./excel-api-requirement-sets.md)
```

Comment:
Is this in the same folder? If so, then the relative path can probably be simplified...
```suggestion
- [Excel JavaScript API requirement sets](excel-api-requirement-sets.md)
```